c10/core/Backend.h

```diff
@@ -20,7 +20,7 @@ namespace c10 {
  * would make sense in your use case.  If it doesn't make sense, maybe
  * you want DeviceType.
  */
-enum class Backend { CPU, CUDA, HIP, SparseCPU, SparseCUDA, SparseHIP, Undefined, NumOptions };
+enum class Backend { CPU, CUDA, HIP, SparseCPU, SparseCUDA, SparseHIP, XLA, Undefined, NumOptions };
```

Comment:
see below, I think we should put TPU in the name (I'm okay if XLA is also in the name).

Reply:
Same discussion as above -- there's a material difference between XLA as a backend and the devices it supports.